app/stac_api/serializers.py

```diff
@@ -913,3 +915,79 @@ def validate(self, attrs):
         validate_json_payload(self)
 
         return attrs
+
+
+class AssetUploadListSerializer(serializers.ListSerializer):
+    # pylint: disable=abstract-method
+
+    def to_representation(self, data):
+        return {'uploads': super().to_representation(data)}
+
+    @property
+    def data(self):
+        ret = super(serializers.ListSerializer, self).data
+        return ReturnDict(ret, serializer=self)
+
+
+class UploadPartSerializer(serializers.Serializer):
+    # pylint: disable=abstract-method
+    etag = serializers.CharField(source='ETag', allow_blank=False, required=True)
+    part_number = serializers.IntegerField(
+        source='PartNumber', min_value=1, max_value=100, required=True, allow_null=False
```

Comment:
Would it make sense to use a global setting or env var for this, as it is also used in `models.py` to ensure consistence?